frontend/src/pages/org/index.ts

```diff
@@ -60,10 +60,7 @@ type Params = {
   new?: ResourceName;
 };
 
-type OrgEventMap = {
-  "execution-minutes-quota-update": QuotaUpdate;
-  "storage-quota-update": QuotaUpdate;
-};
+type OrgEventMap = APIEventMap;
```

Comment:
Do we need this type alias?

Reply:
Nope, this draft PR is still in WIP, just wanted your eyes on the WIP.

Reply:
Oh gotcha, sorry!